src/components/Spawn/IconTooltip.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import Tooltip from "@leafygreen-ui/tooltip";
+import Icon from "components/Icon";
+import { size } from "constants/tokens";
+
+interface IconTooltipProps {
+  tooltipText: string;
+  iconType: string;
+  color: string;
+  shade: string;
+}
+
+export const IconTooltip: React.FC<IconTooltipProps> = ({
```

Comment:
I like this pattern and it will also be used for Projects Track Multiple Branches. We should move this to a  more generic location.
<img width="380" alt="image" src="https://user-images.githubusercontent.com/4605522/159049284-c08c54f4-502e-436f-8930-8c2f7ad6f683.png">